_includes/releases/v23.1/v23.1.1.md

```diff
@@ -0,0 +1,19 @@
+## v23.1.1
+
+Release Date: May 16, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-1-big-fixes">Bug fixes</h3>
+
+- In CockroachDB v23.1.0 and its testing releases, executing [`COPY`](../v23.1/copy-from.html) to a target table that has multiple column families could corrupt the table. If data was copied into a table with existing rows, the data in those rows may be irrecoverable. The data would need to be dropped and re-[copied](../v23.1/copy-from.html) to be encoded correctly. This has now been fixed. See [Technical Advisory 103220](../advisories/a103220.html) for more information. [#103323][#103323]
```

Comment:
The PR link at the end of the line isn’t working for me.

Reply:
Fixed